dependencies-verify-service/src/main/java/com/microsoft/azure/dependencies/verify/Application.java

```diff
@@ -0,0 +1,12 @@
+package com.microsoft.azure.dependencies.verify;
```

Comment:
Since this is a public repo, we'd better add license header.

Reply:
Yes, forgot to add this.